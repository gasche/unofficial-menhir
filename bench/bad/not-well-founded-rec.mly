%start<int> a
%token A
%%

a: b(A) {}

b(X) : b(b(X)) {}
