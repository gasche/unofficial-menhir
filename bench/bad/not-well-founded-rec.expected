File "not-well-founded-rec.mly", line 7, characters 7-8:
Error: Mutually recursive definitions must have the same parameters.
This is not the case for b.
